package.json

```diff
@@ -32,6 +32,8 @@
     "yarn": ">= 1.22.17",
     "pnpm": "please-use-yarn"
   },
-  "dependencies": {},
+  "dependencies": {
+    "swiper": "^9.2.0"
+  },
```

Comment:
`swiper` should not be added to the top level, it is already added in `apps/web`.

Reply:
removed in latest commit